deepchem/models/torch_models/hf_models.py

```diff
@@ -211,6 +217,14 @@ def load_from_pretrained(  # type: ignore
             else:
                 checkpoint = checkpoints[0]
                 data = torch.load(checkpoint, map_location=self.device)
+                # Delete keys of output projection layer (last layer) as the number of
```

Comment:
@arunppsg Can you add these details to the docstring so they are documented in the user docs?

Reply:
added docs